src/swapper.rs

```diff
@@ -136,7 +136,9 @@ impl<'a> Swapper<'a> {
     let options = self.executor.execute(params);
     let lines: Vec<&str> = options.split('\n').collect();
 
-    let pattern = Regex::new(r#"@thumbs-([\w\-0-9]+)\s+(.+)"#).unwrap();
+    //let pattern = Regex::new(r#"@thumbs-([\w\-0-9]+)\s+(.+)"#).unwrap();
+    let pattern =
+      Regex::new(r#"@thumbs-([\w\-0-9]+)\s+"?([\w\d\[\{\(\-\+/\\\.,\|=\)\}\]!@#\$%\^&~`:<>\?;'\*]+)"?$"#).unwrap();
```

Comment:
If someone knows a better way here, please don't hold back.
`.+` is greedy and matches final `"` as well

Reply:
this also means that user can't match for `"` which can be a problem